.github/workflows/tests.yml

```diff
@@ -52,6 +52,7 @@ jobs:
         pytest -v --ignore=tests/sealapi ./tests
     - name: Test with gtest
       run: |
+        cd tenseal/proto && cmake . && make && cd ../../
```

Comment:
Unfortunately, here I'm missing something about cmake builds. For some reason, it doesn't generate the proto headers for the build otherwise.

On the other hand, setup.py generates them just fine, and I cannot spot the actual difference.

Reply:
when I build the tenseal without `BUILD_TEST` argument, the build is successful.
example:
`cmake .`
`make`

we don't have to build `tenseal_proto` manually, but when passing the argument `BUILD_TEST=TRUE`, the build is failed, and we have to build `tenseal_proto` manually.

I think we are missing something in **Unit Tests** section in CMakeLists.txt file.